packages/cms-cli/package.json

```diff
@@ -16,12 +16,14 @@
     "ora": "^4.0.3",
     "readline": "^1.3.0",
     "shelljs": "0.8.3",
-    "update-notifier": "3.0.1"
+    "update-notifier": "3.0.1",
+    "yargs": "15.4.1"
   },
   "engines": {
     "node": ">=8.9.1"
   },
   "bin": {
+    "banjo": "./bin/cli.js",
```

Comment:
This is just a silly codename borrowed from the earlier days and our mascot Banjo.

![image](https://user-images.githubusercontent.com/266036/88428587-2ae21d80-cdc3-11ea-8c1c-84353457a318.png)